RetailCoder.VBE/Inspections/Results/ObsoleteByRefModifierInspectionResult.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using Rubberduck.Common;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.QuickFixes;
+using Rubberduck.Inspections.Resources;
+using Rubberduck.Parsing.Symbols;
+
+namespace Rubberduck.Inspections.Results
+{
+    public class ObsoleteByRefModifierInspectionResult : InspectionResultBase
+    {
+        private readonly Lazy<IEnumerable<QuickFixBase>> _quickFixes;
```

Comment:
Aaah, that's how we did it prior to @Hosch250's "big dig" inspections/quick-fix refactoring. There's no more inspection-specific "result" class, and results are completely decoupled from quick-fixes now. hmm...